ui/robolectric/BasicSample/README.md

```diff
@@ -0,0 +1,20 @@
+# Basic sample for Robolectric
+
+*If you are new to Robolectric, try this sample first.*
+
+This project uses the Gradle build system. You don't need an IDE to build and execute it but Android Studio is recommended.
+
+1. Download the project code, preferably using `git clone`.
+1. Open the Android SDK Manager (*Tools* Menu | *Android*) and make sure you have installed the *Android Support Repository* under *Extras*. (For more Information click [here](http://developer.android.com/tools/testing-support-library/index.html#setup))
```

Comment:
This step is not necessary for Robolectric... it might not be necessary for instrumentation tests now since we're in gMaven @slinzner ?

Reply:
Done